src/xb-string.c

```diff
@@ -178,6 +178,32 @@ xb_string_search (const gchar *text, const gchar *search)
 	return FALSE;
 }
 
+/**
+ * xb_string_search: (skip)
+ * @text: The source string
```

Comment:
This should probably say that `text` is actually a (`NULL` terminated?) array of somethings

Reply:
Fixed.